opengrok-indexer/pom.xml

```diff
@@ -530,11 +534,43 @@ Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
             <groupId>org.apache.ant</groupId>
             <artifactId>ant</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.containers</groupId>
+            <artifactId>jersey-container-servlet</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-jackson</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.ext</groupId>
+            <artifactId>jersey-bean-validation</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
maybe I missed it but how is this used?

Reply:
To check if expiration timer works for messages:

`await().atMost(2, TimeUnit.SECONDS).until(() -> container.getMessages("info").isEmpty());`

because it is a bad practice to use `Thread.sleep(...)` in tests.

Reply:
Ok. Just missed it.
Isn't there something like a dependency only for tests?

Reply:
`<scope>test</scope>` take care of that.

Reply:
I'm blind sorry